conans/client/conanfile/configure.py

```diff
@@ -10,7 +10,7 @@ def run_configure_method(conanfile, down_options, profile_options, ref):
     if hasattr(conanfile, "config_options"):
         with conanfile_exception_formatter(conanfile, "config_options"):
             conanfile.config_options()
-    else:
+    elif conanfile.activate_default_behaviors:
```

Comment:
Don't love the name, but challenging to find a better one. Some ideas:
- ``auto_configuration = True`` 
- ``manage_options = True``

Maybe "configuration" and "options" should be in the var name?

Reply:
I agree. We should think of another shorter one if possible.